From: Michael Tokarev <mjt@tls.msk.ru>
Subject: Enable sigaltstack everywhere
Bug-Debian: http://bugs.debian.org/805826

Enable sigaltstack syscall on all platform.
Upstream agreed to do this after 2.5 release.

--- a/linux-user/syscall.c
+++ b/linux-user/syscall.c
@@ -8294,10 +8294,4 @@ abi_long do_syscall(void *cpu_env, int num, abi_long arg1,
     case TARGET_NR_sigaltstack:
-#if defined(TARGET_I386) || defined(TARGET_ARM) || defined(TARGET_MIPS) || \
-    defined(TARGET_SPARC) || defined(TARGET_PPC) || defined(TARGET_ALPHA) || \
-    defined(TARGET_M68K) || defined(TARGET_S390X) || defined(TARGET_OPENRISC)
         ret = do_sigaltstack(arg1, arg2, get_sp_from_cpustate((CPUArchState *)cpu_env));
         break;
-#else
-        goto unimplemented;
-#endif
 
